lookout/style/format/visualization.py

```diff
@@ -70,9 +73,9 @@ def visualize(input_filename: str, bblfsh: str, language: str, model: str) -> No
 
         new_content += GREEN + CLASSES[wrong.y] + RED + CLASSES[wrong.pred] + ENDC
 
-        if i == len(mispred) - 1 and end != len(old_content):
-            new_content += old_content[end:]
+        if i == len(mispred) - 1:
+            if end != len(old_content):
```

Comment:
@m09 fyi - tricky fix here

Reply:
BTW @EgorBu Would be nice to put a comment here

Reply:
will do in one of the next PRs